app/src/shared/contexts/StreamrClient/index.jsx

```diff
@@ -34,6 +34,7 @@ export function createClient(sessionToken: ?string) {
         },
         autoConnect: true,
         autoDisconnect: false,
+        verifySignatures: 'never',
```

Comment:
Makes canvases work. Without it, a regular `Clock` → `Random number` → `Chart` produce plenty errors about signature validation failures.